### PR TITLE
F sharp unit2

### PR DESCRIPTION
**Pull request by bobbychopra**
Finished updating F# bootcamp for Unit 2

### CODE REVIEW

File: src/Unit-1/FSharp/lesson3/README.md [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+
+As you can see, we're now handing off the input from the console to the `validationActor` for validation and decisions. `consoleReaderActor` is now only responsible for reading from the console and handing the data off to another more sophisticated actor.
+
+There. That's better.
+
+### Once you're done
+Compare your code to the solution in the [Completed](Completed/) folder to see what the instructors included in their samples.
+
+If everything is working as it should, the output you see should be identical to last time:
+![Petabridge Akka.NET Bootcamp Lesson 1.2 Correct Output](Images/working_lesson3.jpg)
```
Comment (skotzko) [inline src/Unit-1/FSharp/lesson3/README.md]: this image is broken

**Comment by skotzko** [comment]
@bobbychopra in general, this is looking great! Several images in these lessons are broken (e.g. lesson 1.3 readme) so I'd just step through the READMEs on GitHub to make sure the images are all good, and then I can merge it in. Great job! :+1:

**Comment by bobbychopra** [comment]
Let me go through them again, thought I had fixed it

**Comment by bobbychopra** [comment]
Sorry about the mess up ... I fixed them. Deleted PR#154, and changes are
in PR#151

**Pull request closed without merging**